Enterprise-CMCS/macpro-mdct-mcr
# [MDCT-2685] Make Numberfields show they are nested under Choicelists

### Description

Numberfields were never actually told what to do despite being given the nested boolean. Lets make sure they've got the proper styling to reflect it!

https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/19439679/388a83fc-9963-4aaf-bd33-0e1ec2a2765f

### Related ticket(s)


MDCT-2685

---
### How to test

Create an MLR Program
Go to the Modal Overlay page (The second page's enter details section)
Scroll to the bottom and click yes on 4.6.1
See the new nesting!

---
### Author checklist
